mappings/net/minecraft/server/command/CloneCommand.mapping

```diff
@@ -8,14 +8,67 @@ CLASS net/minecraft/class_3023 net/minecraft/server/command/CloneCommand
 		ARG 1 commandRegistryAccess
 	METHOD method_13090 execute (Lnet/minecraft/class_2168;Lnet/minecraft/class_3023$class_8010;Lnet/minecraft/class_3023$class_8010;Lnet/minecraft/class_3023$class_8010;Ljava/util/function/Predicate;Lnet/minecraft/class_3023$class_3025;)I
 		ARG 0 source
+		ARG 1 begin
+		ARG 2 end
+		ARG 3 destination
 		ARG 4 filter
+		ARG 5 mode
 	METHOD method_13094 (Lnet/minecraft/class_2168;)Z
 		ARG 0 source
 	METHOD method_13096 (Lnet/minecraft/class_2694;)Z
 		ARG 0 pos
 	METHOD method_13099 (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
 		ARG 0 maxCount
 		ARG 1 count
+	METHOD method_48035 createModeArgs (Lnet/minecraft/class_3023$class_8009;Lnet/minecraft/class_3023$class_8009;Lnet/minecraft/class_3023$class_8009;Lnet/minecraft/class_3023$class_8009;Lcom/mojang/brigadier/builder/ArgumentBuilder;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
```

Comment:
Args or Arguments?

Reply:
I think the method names would get quite long with `arguments`, but it might be more clear. Should probably get more oppositions